data/data/azure/vnet/main.tf

```diff
@@ -50,6 +50,12 @@ resource "azurerm_storage_account" "cluster" {
   account_replication_type        = "LRS"
   min_tls_version                 = contains(local.environments_with_min_tls_version, var.azure_environment) ? "TLS1_2" : null
   allow_nested_items_to_be_public = false
+
+  network_rules {
+    default_action             = "Allow"
+    virtual_network_subnet_ids = [azurerm_subnet.master_subnet[0].id]
+    bypass                     = ["Logging", "AzureServices"]
```

Comment:
I am unsure what options are necessary to bypass if any.
The list of options can be found [here](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/storage_account_network_rules#bypass)

Reply:
They're trusted services so let's just include them all--i.e. add `metrics`